.github/workflows/main.yml

```diff
@@ -36,7 +36,44 @@ jobs:
       uses: coverallsapp/github-action@master
       with:
         github-token: ${{ secrets.GITHUB_TOKEN }}
-        path-to-lcov: ./coverage/lcov.net7.0.info
+
+  PublishGithub:
+    if: github.ref == 'refs/heads/master' && startsWith(github.repository, 'neo-project/')
+    needs: Test
+    runs-on: ubuntu-latest
+    steps:
+    - name: Chectout
+      uses: actions/checkout@v4
+
+    - name: Setup .NET Core
+      uses: actions/setup-dotnet@v4
+      with:
+        dotnet-version: ${{ env.DOTNET_VERSION }}
+
+    - name: Set Version
+      run: git rev-list --count HEAD | xargs printf 'CI%05d' | xargs -I{} echo 'VERSION_SUFFIX={}' >> $GITHUB_ENV
+
+    - name : Pack (Neo)
+      run: |
+        dotnet pack \
+        --configuration Debug \
+        --output ./out \
+        --version-suffix ${{ env.VERSION_SUFFIX }}
+
+    - name: Remove Unwanted Files
+      working-directory: ./out
+      run: |
+        rm -v Neo.CLI*
+        rm -v Neo.GUI*
+
+    - name: Publish to Github Packages
+      working-directory: ./out
+      run: |
+        dotnet nuget push * \
+            --source https://nuget.pkg.github.com/neo-project/index.json \
+            --api-key "${{ secrets.PKGS_TOKEN }}" \
```

Comment:
Can we try with `secrets.GITHUB_TOKEN`? As I understand it require a new secret that must be a owner token, otherwise please MD me on discord.

Reply:
We can try. The problem was more likely me, trying it in a PR.

Reply:
Let's try with `secrets.GITHUB_TOKEN`